docs/wiki/PolicyDeepDive.md

```diff
@@ -70,4 +70,169 @@ Please see the following wiki article that explains this process further: [Addin
 
 ## How do we assign a built-in or custom Azure Policy?
 
-Please see the following wiki article that explains this process further: [Assigning Azure Policies](https://github.com/Azure/ALZ-Bicep/wiki/AssigningPolicies)
\ No newline at end of file
+Please see the following wiki article that explains this process further: [Assigning Azure Policies](https://github.com/Azure/ALZ-Bicep/wiki/AssigningPolicies)
+
+## How to migrate ALZ custom policies to Azure built-in policies
+
+Every so often there will be a new Azure built-in policy or policy initiative that replicates functionality previously found only in ALZ custom policies. When this happens, a new release of ALZ-Bicep will contain details around the policies involved as well as upgrade considerations. The below describes generic scenarios for migrating from ALZ custom policies to Azure built-in policies, leveraging ALZ-Bicep functionality. fixme assuming that ALZ-Bicep is leveraged completely i.e. with default policy assignments.
+
+### Update scenarios
+There are the following scenarios for ALZ custom policies being superseded by Azure built-in policies, listed in order of complexity:
+1. A single ALZ custom policy, which is not assigned anywhere in your Azure estate, is superseded by an Azure built-in policy.
+2. A single ALZ custom policy, which is assigned at one or more scopes in your Azure estate, is superseded by an Azure built-in policy. The process for managing this is described in [Migrate single ALZ custom policy to built-in policy](#migrate-single-pol).
+3. One or more ALZ custom policies, assigned via ALZ custom policy intiative, which are superseded by Azure built-in policies. The process for managing this is described in [Migrate ALZ custom policies in initiatives to built-in policies](#migrate-multiple-pol).
+
+#### Migrate single ALZ custom policy to built-in policy
+For this scenario we will use the ALZ custom policy _Deny the creation of public IP_ which will be migrated to the built-in policy _Not allowed resource types_
+
+To carry out the instructions in the scenario the operator will require Resource Policy Permissions at the root of the ALZ management group hierarchy
+
+- Go to https://portal.azure.com
+- Open Policy
+- Go to Definitions and in Search find the ALZ custom policy.
+
+  ![alz-custom-policy-def-search](media/alz-update-to-builtin-01.png)
+
+- Click on the hyperlink for the policy definition
+- To determine if the policy is assigned at any scope in the ALZ management group structure start by getting the policy definition ID
+
+  ![alz-custom-policy-def-name](media/alz-determine-policy-assign-01.png)
+
+- Since there is no easy way to get the various scopes a policy is assigned to, go to Azure Resource Graph Explorer
```

Comment:
Could you not click on assignments in the definition view blade?

Reply:
I could, but that pressuposes that I choose all the relevant scopes when I select the policy definition. otherwise the assignments may not show. RG Explorer is surefire